time_capture/patches.txt

```diff
@@ -3,5 +3,5 @@
 # Read docs to understand patches: https://frappeframework.com/docs/v14/user/en/database-migrations
 
 [post_model_sync]
-execute:from time_capture.install import after_install;after_install() # 2025-03-19 #2
-time_capture.patches.log_inconsistent_time_capture_logs 
```

Comment:
The `log_inconsistent_time_capture_logs` patch being removed in this PR without further comment is unexpected. In general, patches run only once so in most cases it's not necessary to remove them.